README.md

```diff
@@ -3,10 +3,9 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/facebookresearch/habitat-sim/blob/master/LICENSE)
 [![Anaconda-Server Badge](https://anaconda.org/aihabitat/habitat-sim/badges/version.svg)](https://anaconda.org/aihabitat/habitat-sim)
 [![Anaconda-Server Badge](https://anaconda.org/aihabitat/habitat-sim/badges/platforms.svg)](https://anaconda.org/aihabitat/habitat-sim)
+[![Documentation](https://img.shields.io/badge/docs-automated-green.svg)](https://aihabitat.org/docs/habitat-sim/)
```

Comment:
This should actually be a circleci badge that points to our automatic docs deployment job so we know if docs suddenly start failing.

Reply:
@Skylion007  thank you for the suggestion: I can't find [CircleCI tags](https://circleci.com/docs/2.0/status-badges/) or Shields.io supporting specific step in workflow status represented as the badge. If our CircleCI build badge is green => Docs build is green for sure.

Reply:
Yeah you are right, that's a huge oversight. It shouldn't be automated green then, it should informative blue.